src/liballoc/heap.rs

```diff
@@ -395,6 +425,12 @@ mod imp {
     }
 
     pub fn stats_print() {}
```

Comment:
I would provide a really neat implementation using `HeapSummary`, but I don't know of an easy way to print things from inside this crate.